src/app/features/settings/components/settings-form/settings-form.component.html

```diff
@@ -0,0 +1,95 @@
+<form class="w-1/2" [formGroup]="form" (ngSubmit)="submit()">
+  <p class="text-blue-600 font-bold mt-4">General</p>
+  <p class="form-input-label">Avatar</p>
+
+  <custom-file-upload
+    accept=".jpg,.jpeg"
+    [url]="user().avatar"
+    (upload)="uploadFile($event)"
+  />
+
+  <label class="form-input-label" for="name">Name</label>
+  <input
+    class="form-input"
+    formControlName="name"
+    id="name"
+    placeholder="Name"
+    type="text"
+  />
+  @if (!form.controls['name'].untouched) {
+  <validation-message [errors]="form.controls['name'].errors" />
+  }
+
+  <label class="form-input-label" for="email">Email</label>
+  <input
+    class="shadow disabled form-input !bg-gray-200"
+    formControlName="email"
+    id="email"
+    placeholder="Email"
+    type="email"
+    readonly
+  />
+
+  <div formGroupName="address">
+    <p class="text-blue-600 font-bold mt-8">Address</p>
+    <label class="form-input-label" for="street">Street Address</label>
+    <input
+      class="form-input"
+      formControlName="street"
+      id="street"
+      placeholder="Street Address"
+      type="text"
+    />
+    @if (!form.get('address.street').untouched) {
+    <validation-message [errors]="form.get('address.street').errors" />
+    }
+    <div class="flex">
+      <div class="w-1/3">
```

Comment:
`w-1/3` and similar utils are not a great way to do grids. You have to apply it to all the children. It is better to use `grid` and `grid-cols-x` utils from tailwind.